local-cli/automated-vs/solutions/uwpSolution.js

```diff
@@ -0,0 +1,126 @@
+
+        return new Promise(resolve => {
+            const popd = this.pushd(options.root);
+
+            console.log(chalk.green('Removing old version of the app'));
+            execSync(`powershell -ExecutionPolicy RemoteSigned Import-Module "${windowsStoreAppUtils}" ; Uninstall-App ${appName}`, execOptions);
+
+            console.log(chalk.green('Installing new version of the app'));
+            execSync(`powershell -ExecutionPolicy RemoteSigned Import-Module "${windowsStoreAppUtils}"; Install-App "${script}"`, execOptions);
+
+            const appFamilyName = execSync(`powershell -c $(Get-AppxPackage -Name ${appName}).PackageFamilyName`).toString().trim();
+            execSync(`CheckNetIsolation LoopbackExempt -a -n=${appFamilyName}`, execOptions);
+
+            console.log(chalk.green('Starting the app'));
+            execSync(`powershell -ExecutionPolicy RemoteSigned Import-Module "${windowsStoreAppUtils}"; Start-Locally ${appName} ${args}`, execOptions);
+
+            popd();
+
+            resolve();
+        });
+    }
+
+    deployToDevice(options, alternateWindowsPath) {
+        const appPackageFolder = this.getAppPackageFolder(options.arch, options.configuration, alternateWindowsPath);
+        const deployTarget = options.target ? options.target : (options.emulator ? 'emulator' : 'device');
+        const deployTool = new WinAppDeployTool();
+        const appxManifest = this.getAppxManifest(options.arch, options.configuration, alternateWindowsPath);
+        const identity = appxManifest.root.children.filter(function (x) { return x.name === 'mp:PhoneIdentity'; })[0];
+        const appName = identity.attributes.PhoneProductId;
+
+        return new Promise(resolve => {
+            const device = deployTool.findDevice(deployTarget);
+
+            try {
+                deployTool.uninstallAppPackage(appName, device, options.verbose);
+            } catch (e) {
+                console.log(chalk.yellow('Failed to uninstall app from ' + device.name));
+            }
+
+            const appxFile = glob.sync(path.join(appPackageFolder, '*.appx'))[0];
+            try {
+                console.log(chalk.white(deployTool.installAppPackage(appxFile, device, true, false, options.verbose)));
+                resolve();
+            } catch (e) {
+                if (e.message.indexOf('Error code 2148734208 for command') !== -1) {
+                    console.log(chalk.white(deployTool.installAppPackage(appxFile, device, true, true, options.verbose)));
+                    resolve();
+                } else {
+                    this.handleResponseError(e);
+                }
+            }
+        });
+    }
+
+    //assumes app package is in some AppPackages/*_[arch]_[configuration]_* directory
+    getAppPackageFolder(arch, configuration, alternateAppPackagePath) {
+        const baseAppPackagesPath = `AppPackages/*_${arch}_*` + (configuration === 'Debug' ? `${configuration}_*` : '');
```

Comment:
release previously didn't work, as it looked for the AppPackages in `AppPackages/*_[arch]_[configuration]_*`, but in actuality, the AppPackage Folder is in `AppPackages/*_${arch}_*` if compiled for release, and `AppPackages/*_${arch}_*${configuration}_*` if compiled for debugging